tardis/plasma/properties/ion_population.py

```diff
@@ -136,36 +137,31 @@ class RadiationFieldCorrection():
     This function adds a field 'delta' to the phi table given to the function
 
     """
-
-    def calculate(self, w, departure_coefficient=None, chi_0_species=(20, 2)):
-        #factor delta ML 1993
-        if departure_coefficient is None:
-            departure_coefficient = 1. / self.ws
-
-        ionization_data = self.atom_data.ionization_data
-
+    @staticmethod
+    def calculate(w, ionization_data, beta_rad, t_electron, t_rad,
+        beta_electron, levels):
+        # factor delta ML 1993
+        departure_coefficient = 1. / w
+        chi_0_species=(20, 2)
```

Comment:
Okay for these kinds of things, we want them to be parameters of the object, so put them in the `__init__` and them access them with `self`. Does this make sense?